### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by Nadine2016**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/cloud/ibm/_index.md [hunk 1]
```diff
@@ -52,9 +52,9 @@ To create or prepare your cluster for Portworx:
...
+In order for your Portworx cluster to be highly available, you must ensure that the Portworx key-value store is set up highly available. By using an IBM Cloud Database-as-a-Service, such as [IBM Compose for etcd for IBM Cloud](https://cloud.ibm.com/docs/services/ComposeForEtcd/getting_started.html#getting-started-tutorial) or [Databases for etcd](https://cloud.ibm.com/docs/services/databases-for-etcd/getting-started.html#getting-started-tutorial) you can set up a highly available key-value store for your Portworx cluster. Each service instance contains three etcd data members that are added to a cluster. The etcd data members are spread across zones in an IBM Cloud location and data is replicated across all etcd data members.
 
-Follow the [IBM Cloud Kubernetes Service documentation](https://cloud.ibm.com/docs/containers/cs_storage_portworx.html#portworx_database) to set up your IBM Compose for etcd key-value store for Portworx.
+Follow the [IBM Cloud Kubernetes Service documentation](https://cloud.ibm.com/docs/containers/cs_storage_portworx.html#portworx_database) to set up your IBM Compose or Databases for etcd key-value store for Portworx.
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: The previous message was more clear. IBM Compose acts as the etcd key-value store for Portworx. So the additional  text "or Databases" is confusing.

How about 

```
to set up your IBM Compose as etcd key-value store for Portworx.
```

Comment (Nadine2016) [reply content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: Hi, the problem is that the service is called "Databases for etcd" and "Compose for etcd. So this sentence is just saying ...to set up your Compose for etcd or Databases for etcd key-value store..

Comment (Nadine2016) [reply content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: I know it is a weird name ... :-/

Comment (Nadine2016) [reply content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: over time, we want to replace Compose for etcd with Databases for etcd so we will take out the Compose for etcd part

**Pull request merged**